modular_skyrat/modules/assault_operatives/code/assault_operatives.dm

```diff
@@ -0,0 +1,260 @@
+#define OBJECTIVE_COUNT 5
+
+/**
+ * ASSAULT OPERATIVE ANTAG DATUM
+ */
+
+/datum/antagonist/assault_operative
+	name = ROLE_ASSAULT_OPERATIVE
+	job_rank = ROLE_ASSAULT_OPERATIVE
+	roundend_category = "assault operatives"
+	antagpanel_category = "Assault Operatives"
+	antag_hud_name = "synd"
+	antag_moodlet = /datum/mood_event/focused
+	show_to_ghosts = TRUE
+	hijack_speed = 2
+	preview_outfit = /datum/outfit/syndicate
+	var/assault_operative_default_outfit = /datum/outfit/syndicate
+	var/datum/team/assault_operatives/assault_team
+	/// Should we move the operative to a designated spawn point?
+	var/send_to_spawnpoint = TRUE
+	/// What class have we equipped? None means we haven't selected one yet.
+	var/equipped_class = null
+	//If not assigned a team by default ops will try to join existing ones, set this to TRUE to always create new team.
+	var/always_new_team = FALSE
+	var/spawn_text = "Your mission is to assault NTSS13 and get all of the GoldenEye keys that you can from the heads of staff that reside there. \
+	Use your pinpointer to locate these after you have extracted the GoldenEye key from the head of staff. It will be sent in by droppod. \
+	You must then upload the key to the GoldenEye upload terminal on this GoldenEye station. After you have completed your mission, \
+	The GoldenEye defence network will fall, and we will gain access to Nanotrasen's military systems. Good luck agent."
```

Comment:
Does it need to be a new variable declaration? Is that not already something that every antagonist has?

Reply:
I think it's a TGUI thing, not sure tbh